tests/page-objects/default/common/common.wdio.page.js

```diff
@@ -26,8 +26,8 @@ const getTasksButtonLabel = () => $('#tasks-tab .button-label');
 const getAllButtonLabels = async () => await $$('.header .tabs .button-label');
 const loaders = () => $$('.container-fluid .loader');
 const syncSuccess = () => $('aria/All reports synced');
-const syncInProgress = () => $('*="Currently syncing"');
-const syncRequired = () => $(`${hamburgerMenuItemSelector}.sync-status .required`);
+const syncInProgress = () => $(hamburgerMenuItemSelector).$('*="Currently syncing"');
+const syncRequired = () => $(`${hamburgerMenuItemSelector} .sync-status .required`);
```

Comment:
Sync patterns change  a bit. The old selector could have hit the snackbar (it uses the same text) disappearing instead of the menu item. so this is still correct.